cnf/Local/index.xml.sha

```diff
@@ -1 +1 @@
-00d32998f75fb42424a40b7de9e81e4dc9be2ca6a9fc3f2d3bec406919d62140
\ No newline at end of file
+8de8e8866152096f0c4ea5ddd6ed1d633b4d1a6edfbd823245cb57f6639732fd
```

Comment:
Don't include the sha and files in cnf/local by default... There change is not relevant to anything that needs to be in the repository.  Especially when no bundles changed